loom/common/core/performance.cpp

```diff
@@ -252,8 +244,20 @@ LoomProfilerRoot::LoomProfilerRoot(const char *name)
     mTotalInvokeCount       = 0;
     mFirstLoomProfilerEntry = NULL;
     mEnabled                = true;
+    mTelemetryVisited       = false;
 }
 
+LoomProfilerRoot* LoomProfilerRoot::fromName(const char *name)
```

Comment:
It's how we retrieve the correct profiler root from a function name now. It's possible that it's slow, however not any slower than before as the same check happened in the constructor above.